src/query/storages/fuse/fuse/src/operations/fuse_sink.rs

```diff
@@ -145,6 +150,7 @@ impl Processor for FuseTableSink {
                 output.finish();
             }
             self.state = State::Finished;
+            self.ctx.push_precommit_column_ndvs(self.ndvs.clone());
```

Comment:
If a delete/insert is distributed(for example: insert into t1 select * from t2), how do we calculate the ndv?

Reply:
emm, that is a question, i will check what `push_precommit_block` function has been done and the whole insert\delete process, to see if `push_precommit_column_ndvs ` is collect, wait please.